sql/hive/src/main/antlr3/org/apache/spark/sql/parser/SparkSqlParser.g

```diff
@@ -637,7 +637,7 @@ import org.apache.hadoop.hive.conf.HiveConf;
   // counter to generate unique union aliases
   private int aliasCounter;
   private String generateUnionAlias() {
-    return "_u" + (++aliasCounter);
+    return "u_" + (++aliasCounter);
```

Comment:
Yes. `_u` appears as an alias of a subquery. I hit this issue while trying to fix `HiveQuerySuite.CTE feature #2`.